docs/client.md

```diff
@@ -0,0 +1,53 @@
+# pb-cli specification
+
+## Overview
+
+`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-reading and private share.
```

Comment:
```suggestion
`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-read and private share.
```

fix grammar error.

BTW, use `remark-lint` to make sure it correct somewhat standard to make sure our code style is pretty good.

Reply:
I search on wikipedia, it exactly is "burn after reading".

I use `DavidAnson/markdownlint`. It is a vscode plugin.

Reply:
Fine, do it as you pleased.